cli/printer/text_test.go

```diff
@@ -43,26 +43,30 @@ func ExampleTextPrintEnvironments() {
 			EnvironmentID:   "id1",
 			EnvironmentName: "name1",
 			OperatingSystem: "linux",
-			ClusterCount:    1,
-			InstanceSize:    "m3.medium",
+			MinScale:        1,
+			CurrentScale:    2,
+			MaxScale:        3,
+			InstanceType:    "m3.medium",
 			Links:           []string{"id2"},
 		},
 		{
 			EnvironmentID:   "id2",
 			EnvironmentName: "name2",
 			OperatingSystem: "windows",
-			ClusterCount:    2,
-			InstanceSize:    "m3.xlarge",
+			MinScale:        2,
+			CurrentScale:    5,
+			MaxScale:        50,
+			InstanceType:    "m3.xlarge",
 			Links:           []string{"id1", "api"},
 		},
 	}
 
 	printer.PrintEnvironments(environments...)
 	// Output:
-	// ENVIRONMENT ID  ENVIRONMENT NAME  OS       CLUSTER COUNT  INSTANCE SIZE  LINKS
-	// id1             name1             linux    1              m3.medium      id2
-	// id2             name2             windows  2              m3.xlarge      id1
-	//                                                                          api
+	// ENVIRONMENT ID  ENVIRONMENT NAME  OS       SCALE   INSTANCE TYPE  LINKS
```

Comment:
Seems like `SCALE` should have some sort of key for the colon separated values.

Reply:
Not sure what you mean?

Reply:
How does the user know what each value stands for? (eg. `1:2:50`)

Reply:
how do we do that for service deployments?

Reply:
I'm assuming you mean `loadbalancer get`, which is somewhat similar in that two ports and a protocol are presented. We don't give a ton of help there either, except for the `USAGE` notes for `addport` and `dropport`: e.g. `l0 loadbalancer addport [command options] NAME HOST_PORT:CONTAINER_PORT/PROTOCOL`

Anyway, I think a bit in `Usage`, accessible via `--help` would be good enough for the moment.

Reply:
I actually meant `service get` where it shows something like:
```
SCALE
0/1 (1)
```

Reply:
I believe that's currently explained in docs, in which case I don't see why this would be any different